docs/examples.md

```diff
@@ -340,6 +340,20 @@ boot_device:
         thumbprint: REPLACE-THIS-WITH-YOUR-TANG-THUMBPRINT
 ```
 
+This example uses the shortcut `boot_device` syntax to configure an encrypted root filesystem in s390x on the `dasda` DASD device unlocked with a CEX card.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.6.0-experimental
+boot_device:
+  layout: s390x-eckd
+  luks:
+    device: /dev/dasda
+    cex:
```

Comment:
So we have a 3d object, with only two dementions ? on and off? 

Are there plans for adding other fields to the cex object?

if not we can we make it 2d? i.e `cex: bool `

Reply:
There were few s390x specific parameter that can be selected under cex type. But later those parameter added as default and hardcoded in the ignition and decided that we can add in future if there is a requirement eg CIPHER. That's why the cex.enabled. 

https://github.com/coreos/butane/pull/536#discussion_r1675936862